Docs - Improve SEO (#99)
iotaledger/IOTA-2.0-Research-Specifications: docs/5.2LedgerState.md

---

> description: The Realities Ledger State deals with double spends and leverages on certain properties of UTXO.  

Reviewer: ```suggestion
description: The Realities Ledger State prevents double spending by leveraging certain properties of UTXO.  
```

image: /img/logo/Coordicide_Logo_Black.png
keywords:

> - transactions

Reviewer: ```suggestion
- transactions
- double spending
```
